docs/src/main/asciidoc/_cloud/References.adoc

```diff
@@ -0,0 +1,23 @@
+# References
+
+# Helm
+* https://helm.sh/[Official page]
+* https://helm.sh/docs/intro/install/[Installing Helm]
+* https://github.com/wildfly/wildfly-charts/blob/main/charts/wildfly/README.md[Helm Chart for WildFly]
+** https://www.wildfly.org/news/2021/05/05/helm-charts-for-wildfly/[Related blog post]
+* https://docs.wildfly.org/wildfly-charts/[Helm Charts for WildFly]
+
+# OpenShift
+* https://access.redhat.com/downloads/content/290/ver=4.12/rhel---9/4.12/x86_64/product-software[Download OpenShift]
```

Comment:
This link for OCP 4.12 will be outdated when new versions are released

Reply:
This [link](https://access.redhat.com/downloads/content/290/) seems to give the same result